game-backend/src/helper/gameObjects.ts

```diff
@@ -18,3 +23,41 @@ export const countUnits = (state: State): number =>
 
 export const countFood = (state: State): number =>
   countGameObjectsPerType(state, GAME_OBJECT_TYPES.FOOD);
+
+export const getFreeFields = (state: State) => {
+  const fields: Position[] = new Array(ARENA_WIDTH)
+    .fill(undefined)
+    .reduce(
+      (acc, _, i) => [
+        ...acc,
+        ...new Array(ARENA_HEIGHT)
+          .fill(undefined)
+          .map((omit, j) => ({ x: i, y: j })),
```

Comment:
Why do you use `_` and `omit` as variable names for omitted values? (In case it's just for handling eslint, you can easily configure this rule https://eslint.org/docs/rules/no-shadow#allow)